bundles/CustomReportsBundle/src/Controller/Reports/CustomReportController.php

```diff
@@ -510,4 +486,26 @@ public function isValidConfigName(string $configName): void
             throw new \Exception('The customer report name is invalid');
         }
     }
+
+    /*
+     * gets the sort, direction, filters, drilldownfilters from grid or initial config
+     */
+    protected function getSortAndFilters(Request $request, \stdClass $configuration): array
+    {
+        $sortingSettings = null;
+        $sort = null;
+        $dir = null;
+        if(class_exists('\Pimcore\Bundle\AdminBundle\Helper\QueryParams')) $sortingSettings = \Pimcore\Bundle\AdminBundle\Helper\QueryParams::extractSortingSettings(array_merge($request->request->all(), $request->query->all()));
```

Comment:
```suggestion
    private function getSortAndFilters(Request $request, \stdClass $configuration): array
    {
        $sortingSettings = null;
        $sort = null;
        $dir = null;
        if(class_exists('\Pimcore\Bundle\AdminBundle\Helper\QueryParams')) {
        $sortingSettings = \Pimcore\Bundle\AdminBundle\Helper\QueryParams::extractSortingSettings(array_merge($request->request->all(), $request->query->all()));
        }
```

Reply:
We can make the method private, right? No need for it to be protected.
And SonarCloud is complaining about missing {} for the if statement, I tried to add it in the suggestion, but couldn´t get the formatting right in the browser, sorry.